_env/nginx.conf

```diff
@@ -173,8 +173,21 @@ http {
     rewrite ^/rules                /docs/rules/ permanent;
 
     location / {
-      try_files       $uri.html $uri $uri/ =404;
-      index           index.html index.htm;
+      proxy_pass       https://srobo.github.io/website/;
+      proxy_set_header Host srobo.github.io;
+      # Work around github.io redirect issue (see above)
+      proxy_redirect   http://srobo.github.io/website/ /;
+      proxy_redirect   https://srobo.github.io/website/ /;
+
+      sub_filter "/website/" "/";
+      sub_filter_once off;
```

Comment:
Do we definitely want this off? From my (very limited) understanding and reading the [docs](http://nginx.org/en/docs/http/ngx_http_sub_module.html#sub_filter_once) this would replace every instance of `/website/` with a `/` in a URL. It seems unlikely we'd want a sub directory on the site called `/website/` but if we did I believe this setting would break this. Is this desired?

Reply:
My interpretation is that without this the replacement would only be done one _for the whole document_. I've not tested that and would be happy to drop this if that's not the case though.

Reply:
Just tested this locally, it does seem to be the case that `sub_filter_once on` means replacing one instance in the whole document while `sub_filter_once off` means doing all replacements.

Just for clarity here, there is an alternative option where we set the underlying Jekyll generation to remove the `/website/` portion, however that would mean that anyone who ended up at the GitHub Pages site would find it rather broken. Since we do currently have bugs which mean that that can happen, I figured it was worth the trade-off for now.

Reply:
👍 Thanks for looking into this